charts/lakefs/templates/ingress.yaml

```diff
@@ -55,5 +55,20 @@ spec:
               servicePort: {{ $svcPort }}
           {{- end }}
           {{- end }}
+          {{- range .pathsOverrides }}
+          - path: {{ .path }}
+          {{- if semverCompare ">=1.19-0" $.Capabilities.KubeVersion.GitVersion }}
+            pathType: Prefix
+            backend:
+              service:
+                name: {{ .serviceName }}
+                port: 
+                  number: {{ .servicePort }}
+          {{- else }} 
+            backend:
+              serviceName: {{ .serviceName }}
+              servicePort: {{ .servicePort }}
+          {{- end }}
```

Comment:
Please document this - probably in a comment on each branch of the `if`.  Comment might just link to docs, of course.

Reply:
Usually you'd want to avoid comments in the template itself, added it to the default `values.yaml`